Libplanet/Store/IStateStore.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Immutable;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Blockchain;
+using Libplanet.Blocks;
+
+namespace Libplanet.Store
+{
+    /// <summary>
+    /// An interface to store states.
+    /// </summary>
+    public interface IStateStore
+    {
+        /// <summary>
+        /// Sets states mapped as relation <see cref="Block{T}.Hash"/> → states.
+        /// It guarantees <see cref="GetState"/> will return the same state if you passed same
+        /// <paramref name="blockHash"/> and haven't written overwrite the value, storage was
+        /// not broken.
```

Comment:
```suggestion
        /// <paramref name="blockHash"/> unless it has overwritten.
```

The storage-level breakage does not affect to only this, but almost every function of this type… so I believe such case doesn't have to be mentioned.